### PR TITLE
Add sidebar and display filters

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

File: app/dashboard/sidebar/filterOption/filterOption.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="filter-option">
+    <div class="square">
+        <svg><rect class="{{class}}" /></svg>
```
Comment (csscottlogic) [inline app/dashboard/sidebar/filterOption/filterOption.html]: I'm a bit surprised to see SVG here?

File: app/dashboard/sidebar/filterOption/filterOption.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+	function filterOption() {
+        var directive = {
+            scope: {
+                class: "@",
+                label: "@"
+            },
+            templateUrl: "dashboard/sidebar/filterOption/filterOption.html"
+        };
+		return directive;
```
Comment (csscottlogic) [inline app/dashboard/sidebar/filterOption/filterOption.js]: Indenting seems to have gone a bit wrong in this file?

File: app/dashboard/sidebar/sidebar.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = function (module) {
+    module.directive("sidebar", sidebar);
+
+	function sidebar() {
+        var directive = {
+            templateUrl: "dashboard/sidebar/sidebar.html"
+        };
+		return directive;
```
Comment (csscottlogic) [inline app/dashboard/sidebar/sidebar.js]: Indenting again?

**Comment by csscottlogic** [comment]
Looks good. I guess the ```<svg>``` calls a spade a spade - it is a rectangle. I'd have used a div for simplicity, but I'm not going to argue.
Some of the indenting has gone a bit awry, but apart from that :shipit:

**Comment by jebbinSCL** [comment]
I just looked at the indentation on my machine, and it looks correct. Could it be an issue between spaces and tabs?

**Comment by jebbinSCL** [comment]
Other than indentation :shipit:

**Pull request merged**